src/core/client/stream/test/__snapshots__/loadMore.spec.tsx.snap

```diff
@@ -0,0 +1,210 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Perhaps we can add a `.gitattributes` file where we set:

```
__snapshots__ binary
```

To prevent git from trying to resolve merge conflicts in snapshots?

Reply:
It's fine, resolving merge conflicts is one of the solutions recommended by `jest`.